docs/sources/installation/sizing/index.md

```diff
@@ -26,10 +26,10 @@ This tool helps to generate a Helm Charts `values.yaml` file based on specified
   </select>
 
   <label>Ingest<i class="fa fa-question" v-on:mouseover="help='ingest'" v-on:mouseleave="help=null"></i></label>
-  <input v-model="ingest" name="ingest" placeholder="Desired ingest in GiB/day" type="number" max="1048576" min="0"/>
+  <input style="width: 95%; padding-right:4.5em;" v-model="ingestInGB" name="ingest" placeholder="Desired ingest in GB/day" type="number" max="1048576" min="0"/><span style="margin-left:-4em;">GB/day</span>
```

Comment:
I'm no pro in CSS. I wish I could find out how ti display the input on the full length.

Reply:
I typically use flexbox for all my CSS layout things: https://css-tricks.com/snippets/css/a-guide-to-flexbox/